web/packages/shared/components/TextEditor/TextEditor.jsx

```diff
@@ -104,20 +111,59 @@ class TextEditor extends React.Component {
 
   render() {
     const { bg = 'levels.sunken' } = this.props;
+    const hasButton = this.props.copyButton || this.props.downloadButton;
+
     return (
       <StyledTextEditor bg={bg}>
         <div ref={e => (this.ace_viewer = e)} />
+        {hasButton && (
+          <ButtonSection>
+            {this.props.copyButton && (
+              <EditorButton
+                test-id
+                title="Copy to clipboard"
+                onClick={() => copyToClipboard(this.editor.session.getValue())}
+              >
+                <Copy size="18" />
```

Comment:
Personally I find these buttons too large, the font size behind them is much smaller. This combination doesn't look good to me.